source/documentation/deploying_services/database_services.md

```diff
@@ -1,41 +1,45 @@
-## Using PostgreSQL
+## Using Database Services
 
-GOV.UK PaaS enables you to create a PostgreSQL database service (powered by Amazon Web Services) and bind it to your app.
+GOV.UK PaaS enables you to create a database service and bind it to your app.
 
 In Cloud Foundry, each service may have multiple plans available with different characteristics.
 
-Currently, GOV.UK PaaS offers a ``postgres`` service with multiple plans available.
+Currently, GOV.UK PaaS offers a service for PostgreSQL, MySQL and MongoDB, with multiple plans available. Please note that MongoDB is currently only available through private beta.
 
 To see the available plans, run:
 
 ```
-cf marketplace -s postgres
+cf marketplace -s SERVICE
 ```
 
-Here is a shortened example of the sort of output you will see (the exact plans will vary):
+Where SERVICE is the service you want; it can currently take the following values:
+ * postgres
+ * mysql
+ * mongodb
```

Comment:
I think `*` is normally fine for lists in Markdown. The problem may be that it needs a blank line before the list.